provisioner/roles/manage_ec2_instances/templates/student_inventory/instances_rhel.j2

```diff
@@ -1,30 +1,118 @@
 [all:vars]
 ansible_user=student{{ item }}
 ansible_ssh_pass={{ admin_password }}
-{% if ssh_port is defined %}
+{% if ssh_port is defined -%}
 ansible_port={{ ssh_port }}
 {% endif %}
 
-[web]
-{% for vm in node1_node_facts.instances %}
-{% if 'student' + item == vm.tags.Student %}
-{{ vm.tags.short_name }} ansible_host={{ vm.public_ip_address }}
+[node]
```

Comment:
If it recall, we were leaving `instances_rhel` as is and adding any changes or centos nodes to `instances_smart_mgmt`